### PR TITLE
button updates

### LINKED ISSUE #505: [Button] design have evolved
the Button component, one of the oldest needs update to match the designs

- check each button type from figma
- check fonts, padding, color, borders

### PR DESCRIPTION
**Pull request by akinolu52**
1. closes #505 
2. warning button 
3. success button
4. error button
5. ghost button
6. link button
7. text button

Screenshot 2022-04-29 at 16 01 16
Screenshot 2022-04-29 at 16 00 56

Screenshot 2022-04-29 at 16 01 34

Screenshot 2022-04-29 at 16 01 28
Screenshot 2022-04-29 at 16 01 23

Screenshot 2022-04-29 at 16 01 40

### CODE REVIEW

File: src/components/Button/styles/inititalStyles.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ export const transparent = css`
 export const hoverEffect = css`
     :hover {
         :after {
-            background-color: ${getShade('dark', 10)};
+            /* background-color: ${getShade('dark', 10)}; */
```
Comment (rayyan224) [inline src/components/Button/styles/inititalStyles.ts:72]: Remove this all together, checked the code not needed

**Review by rayyan224 — APPROVED** [review]
Nice work !

**Comment by rayyan224** [comment]
There are a couple of bugs
There's no border  on warning
ghost fully disappears
link has an square border arround on click




[6:52](https://moralisweb3.slack.com/archives/D039X4294PP/p1651272778367729)
ghose fully dissapers
white_check_mark
eyes
raised_hands





[6:53](https://moralisweb3.slack.com/archives/D039X4294PP/p1651272807527189)
link has an square border arround on click

**Comment by rayyan224** [comment]
Hey Warning is still broken,
there's no effect on hover
Screen Shot 2022-04-30 at 5 59 49 PM

File: src/components/Button/styles/inititalStyles.ts (hunk 1 above)
Comment (akinolu52) [reply src/components/Button/styles/inititalStyles.ts:72]: ok

**Pull request merged**